src/translations/en.json

```diff
@@ -1195,7 +1195,10 @@
             "addon_info": "{addon} Info"
           }
         },
-        "filter_placeholder": "Search entities",
+        "filter_placeholder": {
```

Comment:
We should be careful making changes like this, already translated languages will now overwrite the `filter_placeholder` object with a string.

Reply:
Better change this back and make two single strings?

Reply:
I removed the old translations from Lokalise already

Reply:
Ah okay, thnx 👍 Will keep this in mind for the next time!

Reply:
Don't we still have to change this back? Otherwise this would break the translations for the current RC, as this feature is not released in this version.

Reply:
yes, it will break the current release indeed... Can you change it back?

Reply:
Sure 🙂

Reply:
https://github.com/home-assistant/frontend/pull/23131